README.md

```diff
@@ -107,3 +107,9 @@ cd ..
 ```
 
 Now you should be able to start neovim and use cursorless.
+
+#### Configuration Notes
+
+Notes for Linux installs:
+- Some Linux package managers ship with a version of nvim too old for Lazy. If this is the case, [install nvim](https://github.com/neovim/neovim/blob/master/INSTALL.md) via another method
+- Do not use the snap package of nvim, as this will interfere with rpc functionality 
```

Comment:
Do you know why it interferes with rpc functionality? Or is the problem that nvim can't access the globally installed npm packages?

Reply:
That is my assumption but I could be wrong. I will get rid of the qualifier and just say don't use the snap package since without looking into the code I can't really know. Could just a path related issue.

Reply:
This is "node installed with snap" and not "neovim installed with snap" but interesting https://youtrack.jetbrains.com/issue/WEB-35916/npm-not-detected-if-node-installed-via-snap

Reply:
Can just say that "it might not find globally installed neovim npm package"

Reply:
Oh interesting that might be it not sure, I believe my npm version is via snap

```
host@host ~> snap list | grep node
node                                   20.12.2                     8522   20/stable        iojs**          classic
```